index.html

```diff
@@ -1,123 +1,141 @@
-				            of our innovation.</strong></p>
-						</div>
-					</section>
-
-				<!-- Section -->
-					<section id="first">
-						<header>
-							<h2>Its time to work together</h2>
-						</header>
-						<div class="content">
-							<p>Catalyst Biosummit will bring together synthetic biologists and policymakers, academics
-					            and biohackers, and a broad range of professionals invested in biosecurity for a day of
-					            collaborative problem-solving. We aim to catalyze a diverse community of forward-looking
-					            individuals who will <strong>work together to engineer a future enhanced by biology and not
-					            endangered by it.</strong>
-							</p>
-						</div>
-					</section>
-
-					<section id="first">
-						<header>
-							<h2>About the event</h2>
-						</header>
-						<div class="content">
-							<p>
-								This is an experimental event that hopes to create a context for <strong>sharing perspectives, building connections, and sparking collaboration.</strong> Expect a mixture of lightning talks, workshops, and structured group problem-solving, as well as space and support for attendees to convene their own meetups and brainstorming sessions.
-
-							</p>
-						</div>
-					</section>
-
-					<section id="first">
-						<header>
-							<h2>The organizing team</h2>
-						</header>
-						<div class="content">
-							<p>Catalyst is being organized by a team of PhD students, early-career researchers, and biotechnology professionals, including the organizers behind the <a href="https://www.eastbaybiosecurity.org/">East Bay Biosecurity Group</a>. The conference is supported by a grant from the Effective Altruism <a href="https://app.effectivealtruism.org/funds/far-future">Long Term Future Fund</a>. We are being advised by <a href="https://cisac.fsi.stanford.edu/people/megan_palmer">Dr. Megan Palmer</a> and <a href="http://www.sculptingevolution.org/">Dr. Kevin Esvelt</a> and <strong>are currently seeking additional members for our advisory board.</strong> <a href="mailto:eastbaybiosec@gmail.com">Contact us</a> if you're interested.
-							</p>
-						</div>
-					</section>
-
-				<!-- Copyright -->
-					<<div class="copyright">&copy; Theme made by: <a href="https://html5up.net">HTML5 UP</a>.</div>
-
-			</div>
-
-		<!-- Scripts -->
-			<script src="assets/js/jquery.min.js"></script>
-			<script src="assets/js/jquery.scrolly.min.js"></script>
-			<script src="assets/js/browser.min.js"></script>
-			<script src="assets/js/breakpoints.min.js"></script>
-			<script src="assets/js/util.js"></script>
-			<script src="assets/js/main.js"></script>
-
-	</body>
+<!DOCTYPE HTML>
+<!--
+	Paradigm Shift by HTML5 UP
+	html5up.net | @ajlkn
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<!--
+Menu adapted from: https://www.jqueryscript.net/menu/Smart-Sticky-Responsive-Navigation-With-jQuery-CSS3.html
+-->
+<html>
```

Comment:
Ditto this diff, like is :octocat: just having a tough night, or

Reply:
(apparently there were mixed tabs and spaces in the past, for we do not live in the best of all possible worlds)